src/preprocessing/passes/real_to_int.cpp

```diff
@@ -137,11 +136,6 @@ Node RealToInt::realToIntInternal(TNode n, NodeMap& cache, std::vector<Node>& va
             Trace("real-as-int") << "   " << n << std::endl;
             Trace("real-as-int") << "   " << ret << std::endl;
           }
```

Comment:
What happens if you have an array from real to real, and then you have a selection term with a real. Do you change the selection variable to Int? Shouldn't you then convert the array to be from Int to Int? Similarly to UF and other "containers".

Reply:
Int is a subtype of Real in CVC4, so changing Real -> Int is always safe.  No change should be necessary.